integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -279,6 +280,7 @@ public final void before() throws Exception {
     mFsMaster =
         mLocalTachyonClusterResource.get().getMaster().getInternalMaster().getFileSystemMaster();
     mMasterTachyonConf = mLocalTachyonClusterResource.get().getMasterTachyonConf();
+    TTLBucket.setTTlIntervalMs(mMasterTachyonConf.getLong(Constants.MASTER_TTLCHECKER_INTERVAL_MS));
```

Comment:
We don't consult TachyonConf every time because that would expose an API that can create an inconsistent bucket state in production (changing the value of TachyonConf while there are non-empty buckets). I'm definitely on board with replacing the setter with WhiteBox. We should standardize on only using WhiteBox to modify the interval. In the longer term we should make TachyonConf immutable in production and modify it in tests via WhiteBox.